modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs

```diff
@@ -27,6 +32,11 @@ public static class LegacyCliConfigKeys {
         /// </summary>
         public const string PublisherNodeConfigurationFilename = "PublishedNodesFile";
 
+        /// <summary>
+        /// Key for the specified published nodes schema filename.
+        /// </summary>
+        public const string PublisherNodeConfigurationSchemaFilename = "PublishedNodesSchemaFile";
```

Comment:
Please change `PublisherNode` part to `PublishedNodes`.

Reply:
Thanks, we tried to be consistent with the existing one above it i.e. "PublisherNodeConfigurationFilename", wonder if we should update that to be "PublisherNode**s**ConfigurationFilename" as well?

Reply:
Yes, please change the name of that one as well.

Reply:
Name changes will be pushed in next commit.

Reply:
@suneetnangia, I see that you added `s` after `PublisherNode`. Can you also change `Publisher` to `Published` (with `ed` in the end) please ? I now understand now that those two look too similar to each other and I should have called it out in the first comment. If you want to, I can also push this comment, just let me know.

Reply:
Np, a fix coming in next commit.